versions.tf

```diff
@@ -0,0 +1,9 @@
+## Terraform Requirements
+terraform {

```

Comment:
I wonder if we shouldn't add a required version for terraform as well ">=0.13.6" per Abdelhaks example 
```
terraform {
  required_version = ">= 0.13.6"
  required_providers {
    aws        = ">= 3.40.0"
    local      = ">= 1.4"
    kubernetes = ">= 1.11.1"
    http = {
      source  = "terraform-aws-modules/http"
      version = ">= 2.4.1"
    }
  }
}
```